CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+## 0.0.1 -
+
+### Added
+  - lib/schemas/payment.js with validators for processing and creating payments.
+  - lib/index.js which sets up the configurable payment endpoints.
+  - Adds a route GET basePath/credentials for payment api keys / auth.
```

Comment:
We can do without the "basePath/" in the changelog.

Reply:
I can remove it but the basePath is actually configurable. You can change it.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-payment-http/pull/1/commits/092fc2c5a7a8331f0151df314ccbe528d4329cd3